WebApplication/Backend/Repositorys/BuildingRepository.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace WebApplication.Backend.Repositorys
 {
-    public class BuildingRepository
+    public class BuildingRepository : IBuildingRepository
```

Comment:
Do we need repository duplicates?

Reply:
Repositories are duplicated because of current problem with core. This way, the web team and gea team can use the same entities. Hopefully, this problem will be resolved soon and then we can remove duplicates.

Reply:
Okay, then don't forget to do the same changes for both 😄